src/App.tsx

```diff
@@ -1,61 +1,29 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Form } from './components/Form/Form';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { getUserById } from './services/user';
+
+export const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
Ensure that the `id` for new todos is calculated as the largest existing `id` in the array plus one, as specified in the task description. This is important for maintaining unique identifiers for each todo.